files/en-us/web/accessibility/aria/roles/option_role/index.html

```diff
@@ -0,0 +1,107 @@
+---
+title: 'ARIA: option role'
+slug: Web/Accessibility/ARIA/Roles/option_role
+tags:
+  - ARIA
+  - Accessibility
+---
+
+<p>The <code>option</code> role is used for selectable items in a <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role">listbox</a>.</p>
+
+<h2 id="Description">Description</h2>
+
+<p>The <code>option</code> role is used to identify selections a user can make in a <a href="/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role">listbox</a>. These options are similar to the {{HTMLElement('option')}} elements in a {{HTMLElement('select')}} element, but they can contain images.</p>
+
+
+<p>All selected options should have <code>aria-selected</code> set to <code>true</code> and all options that are not selected should have <code>aria-selected</code> set to <code>false</code>. If an option is not selectable, <code>aria-selected</code> should be omitted entirely.</p>
+
+<p>The <code>option</code> role should only be used to identify selections in a <code>listbox</code> so that they can be understood properly by accessibility tools. They must also have an accessible name as the content between its tags or through ARIA with <a href="/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute">aria-label</a> or <a href="/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute">aria-labelled-by</a></p>
```

Comment:
Suggested revisions for this text, along with typo fix for `aria-labelledby`, and make reference to wcag 2.5.3 for situations where devs have a visible label for an option, but then also use aria to provide an accessible name.

```suggestion
<p>The <code>option</code> role is for identifying selectable choices of a <code>listbox</code>. Options must be provided an accessible name. Generally, the accessible name for an option should come from the element's descendant content. Authors can also explicitly provide an accessible name by specifying <a href="/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute">aria-label</a> or <a href="/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute">aria-labelledby</a> to the element with the <code>option</code> role. If using <code>aria-label</code> or <code>aria-labelledby</code>, and the option also displays a visible text label, authors must ensure they adhere to <a href="https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html">WCAG Success Criterion 2.5.3 Label in Name</a>.</p>
```